WordPress/src/main/java/org/wordpress/android/ui/posts/PostListFragment.kt

```diff
@@ -105,25 +105,34 @@ class PostListFragment : Fragment() {
                     }
                 }
 
-                recyclerView?.scrollToPosition(0)
-                postListAdapter.updateItemLayoutType(layoutType)
+                if (postListAdapter.updateItemLayoutType(layoutType)) {
+                    recyclerView?.scrollToPosition(0)
+                }
+            }
+        })
+
+        mainViewModel.authorSelectionUpdated.observe(this, Observer {
```

Comment:
This feels odd because it makes `viewModel` sort of dependent on `PostListFragment`. I wonder if there's a way to let `viewModel` observe `mainViewModel.authorSelectionUpdated` itself. Connecting parent view models with children is hard. ¯\\\_(ツ)\_/¯

Reply:
hmm I didn't look at it this way, but I see your point. I feel like it's quite similar to listening to user's interactions in view and propagating them to the `viewModel`. In this case the `authorSelectionUpdated` isn't always dependant on user interaction but the dependency seems similar. Wdyt?

Reply:
> I feel like it's quite similar to listening to user's interactions in view and propagating them to the viewModel.

Yeah, you're right. Thanks for sharing that. I just wanted to mention that it felt odd to me but I don't think it needs to be changed. 😬

Reply:
>I wonder if there's a way to let viewModel observe mainViewModel.authorSelectionUpdated itself. 

The `viewModel` is a LifeCycle owner so I guess we could pass the MainViewModel into the PostListViewModel and observe the changes directly. But imho it feels a bit hacky. Wdyt?